_component_code/footer.md

```diff
@@ -9,3 +9,57 @@ collection: component_code
 ---
 
 {% include todo.html content="Set up code." %}
+
+# Footer
+
+The footer lives at the bottom of the page to help visitors find
+globally applicable, but non-primary navigation information. We
+recommend using it on every page and every wrapped application. For
+customer applications, it  should not exceed the articulated scope.
+
+## When to use footers
+
+Use the Rackspace footer on every application page to provide consistent
+global functionality on all of our platforms.
+
+## Best practices for footers
+
+Ensure the footer is placed at the bottom of the page, and flush to the right
+of any left-hand navigation, spanning the length of the screen. Ensure the
+footer contains the following:
+
+* Rackspace copyright and year
+* A link to website terms and conditions
+* A link to the privacy policy
+
+# Specifications
+
+## Footer scope
+
+Place the footer at the bottom of the navigational page.
```

Comment:
The following section seems to duplicate information above. Recommend combining and rewriting.